### DISCUSSION TITLE
How to regex match method in intercept?

### DISCUSSION THREAD

**Discussion by aleksandar-b** [cypress-io/cypress#14970] [Questions and Help] [answered]
I want to intercept all requests that are not method OPTIONS.
 
  cy.intercept({ url: '*', method: /^((?!options).)*$/, headers: { 'content-type': 'application/json' } }).as('submit');

**Comment by jennifer-shehane** [comment, marked as answer]
`method` accepts a RegExp arguments, so any regex that is passed that does not match options should work. Try using https://regex101.com/ to come up with an appropriate RegExp.
